src/hundred_pushups/core.cljc

```diff
@@ -168,3 +168,9 @@
 
 (defn format-whitelist-row [row]
   (str (name(first row)) ": " (clojure.string/join "-" (last row))))
+
+(def not-nil? (complement nil?))
```

Comment:
This is fine, but just FYI, there is a function called [some?](https://clojuredocs.org/clojure.core/some%3F) for this - not your fault, clojure's set of functions around truthiness (and checking for truthiness in a collection) have batshit crazy naming.